American Whitewater/Extensions/Reach.swift

```diff
@@ -101,18 +101,44 @@ extension Reach: MKAnnotation {
         }
     }
 
+    public var subtitle: String? {
+        if let sub = section {
+            return sub
+        } else {
+            return ""
+        }
+    }
+    
     public var coordinate: CLLocationCoordinate2D {
         guard let lat = putInLat, let latitude = Double(lat),
             let lon = putInLon, let longitude = Double(lon) else {
-                print("\(id) has invalid coordinates")
+                //print("1. \(name ?? "n/a") \(id) has invalid coordinates (\(putInLat ?? "?"), \(putInLon ?? "?"))")
                 return kCLLocationCoordinate2DInvalid
         }
         let coordinate = CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
         if CLLocationCoordinate2DIsValid(coordinate) {
             return coordinate
         } else {
-            print("\(id) has invalid coordinates")
+            //print("2. \(name ?? "n/a") \(id) has invalid coordinates (\(putInLat ?? "?"), \(putInLon ?? "?"))")
             return kCLLocationCoordinate2DInvalid
         }
     }
+    
+    public var imageName: String {
```

Comment:
If I'm reading this correctly, isn't this a display concern, not a model concern? Seems like it belongs with view code.

Reply:
Agreed.